src/Counter.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class Counter extends React.Component {
+  state = {
+    count: 0,
+  }
+
+  addOne = () => {
+    this.setState(state => ({
+      count: state.count + 1,
+    }));
+  };
+
+  add100 = () => {
+    this.setState(state => ({
+      count: state.count + 100,
+    }));
+  };
+
+  increase = () => {
+    if (this.state.count % 5 === 0) {
+      this.addOne();
+      this.add100();
+    } else {
+      this.addOne();
+    }
+  };
```

Comment:
```suggestion
  increase = () => {
    if (this.state.count % 5 === 0) {
      this.add100();
    } 
    
    this.addOne();
  };
```
You can get rid of `else` here